metaflow/datatools/s3.py

```diff
@@ -922,6 +972,8 @@ def _one_boto_op(self, op, url, create_tmp_file=True):
                     raise MetaflowS3NotFound(url)
                 elif error_code == 403:
                     raise MetaflowS3AccessDenied(url)
+                elif error_code == 601:
+                    raise MetaflowS3InvalidRange(err)
```

Comment:
Isn't the error code - 416? https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#RESTErrorResponses

Reply:
not a big deal since it's an internal detail but to keep it consistent with S3's interface we may want to use 416

Reply:
I'll change. Ya, it was internal but I can use the same one. I always get lost in AWS documentation.